libraries/tock-cells/src/volatile_cell.rs

```diff
@@ -1,81 +1,93 @@
-//! Implementation of a type for accessing MCU registers.
+//! A type for accessing MMIO registers. `VolatileCell` is just like [`Cell`]
+//! but with [volatile] read / write operations
+//!
+//! [`Cell`]: https://doc.rust-lang.org/std/cell/struct.Cell.html
+//! [volatile]: https://doc.rust-lang.org/std/ptr/fn.read_volatile.html
```

Comment:
This should probably say where the code was vendored from and under which license it was placed originally.

Reply:
I added a comment indicating where the code was vendored from. A license header will need to wait for #3318.